File: examples/native/android/app/src/main/java/com/native63/PerformancePlugin.java
package com.native63;

import androidx.annotation.NonNull;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;

import java.util.HashMap;
import java.util.Map;

public class PerformancePlugin extends ReactContextBaseJavaModule {
    PerformancePlugin(ReactApplicationContext context) {
        super(context);
    }

    @NonNull
    @Override
    public String getName() {
        return "PerformancePlugin";
    }

    @NonNull
    @Override
    public Map<String, Object> getConstants() {
        HashMap<String, Object> hm = new HashMap<>();
        hm.put("appInitTimestamp", MainApplication.initTime.getTime());
        return hm;
    }
}

File: examples/native/android/app/src/main/java/com/app/PerformancePlugin.kt
package com.app

import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.ReactContextBaseJavaModule
import com.app.MainApplication

class PerformancePlugin(reactContext: ReactApplicationContext) :
    ReactContextBaseJavaModule(reactContext) {

    override fun getName(): String = "PerformancePlugin"

    override fun getConstants(): Map<String, Any> = hashMapOf("appInitTimestamp" to MainApplication.initTime.time)
}
